src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/URLValidation.feature.disabled

```diff
@@ -0,0 +1,385 @@
+Feature:
+
+        @ignore
+        Scenario: 01 - Ensure clients cannot retrieve information if part of the endpoint is missing
+             When a GET request is made to "/schools"
+             Then it should respond with 404
+              And the response body is
+                  """
+                  {
+                      "detail": "The specified data could not be found.",
+                      "type": "urn:ed-fi:api:not-found",
+                      "title": "Not Found",
+                      "status": 404,
+                      "correlationId": null
+                  }
+                  """
+              And the headers contain
+                  """
+                  Content-Type: application/problem+json
+                  """
+
+        @ignore
+        Scenario: 02 - Ensure clients cannot create a resource if part of the endpoint is missing
+             When a POST request is made to "/schools" with
+                  """
+                  {
+                      "educationOrganizationCategories": [
+                          {
+                              "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#School"
+                          }
+                      ],
+                      "gradeLevels": [
+                          {
+                              "gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Transitional Kindergarten"
+                          }
+                      ],
+                      "schoolId": 2244668800,
+                      "nameOfInstitution": "Institution Test"
+                  }
+                  """
+             Then it should respond with 404
+              And the response body is
+                  """
+                  {
+                      "detail": "The specified data could not be found.",
+                      "type": "urn:ed-fi:api:not-found",
+                      "title": "Not Found",
+                      "status": 404,
+                      "correlationId": null
+                  }
+                  """
+              And the headers contain
```

Comment:
I think we can reuse the existing method 

`And the response headers includes...`

Reply:
Yes, very important to re-use the prior format.

Reply:
Change to match the format